Subscriber/ApplePayDirectSubscriber.php

```diff
@@ -3,21 +3,60 @@
 namespace MollieShopware\Subscriber;
 
 use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Util\Debug;
+use Doctrine\DBAL\Connection;
+use Doctrine\ORM\NoResultException;
 use Enlight\Event\SubscriberInterface;
+use Enlight_Config;
+use Enlight_Controller_Front;
+use Enlight_Controller_Request_RequestHttp;
 use Enlight_Event_EventArgs;
 use Enlight_View;
+use MollieShopware\Components\Account\Account;
 use MollieShopware\Components\ApplePayDirect\ApplePayDirectHandler;
 use MollieShopware\Components\ApplePayDirect\ApplePayDirectHandlerInterface;
 use MollieShopware\Components\ApplePayDirect\ApplePayDirectSetup;
+use MollieShopware\Components\ApplePayDirect\Services\ApplePayPaymentMethod;
 use MollieShopware\Components\Config;
 use MollieShopware\Components\Constants\PaymentMethod;
 use MollieShopware\Components\Constants\ShopwarePaymentMethod;
 use MollieShopware\Components\Logger;
 use MollieShopware\MollieShopware;
+use Shopware\Components\Model\ModelManager;
 use Shopware\Components\Theme\LessDefinition;
+use Shopware\Models\Customer\Customer;
+use Shopware\Models\Order\Order;
+use Shopware_Components_Config;
+use Shopware_Components_Modules;
+use Throwable;
 
 class ApplePayDirectSubscriber implements SubscriberInterface
 {
+    /**
+     * @var null|\Shopware_Components_Modules
+     */
+    private $modules = null;
+
+    /**
+     * @var Account $accountService
+     */
+    private $accountService;
+
+    /**
+     * @var ApplePayPaymentMethod
+     */
+    private $paymentMethodService;
+
+    /**
+     * @param Shopware_Components_Modules|null $modules
+     */
+    public function __construct(Account $accountService, ApplePayPaymentMethod $paymentMethodService, $modules = null)
```

Comment:
hi
could you try either try it with this

` <service id="modules" synthetic="true" />`

or just acessing the Shopware()->Modules in the constructor or where existing...
but we would need to get rid off the nullable object in the contructor
        
thank you

Reply:
I've simply moved the methods to another subscriber because they are not only ApplePayDirect related. So we won't need that nullable arg anymore.